Apromore-Custom-Plugins/Merge-Portal-Plugin/src/main/java/org/apromore/plugin/merge/portal/MergePlugin.java

```diff
@@ -111,6 +111,11 @@ public String getGroupLabel(Locale locale) {
         return "Redesign";
     }
 
+    @Override
+    public String getIconPath() {
+        return "merge/icon.svg";
+    }
+
```

Comment:
What happens if two different plugins have the same resource string, e.g. "merge/icon.svg", but the resource is actually different, e.g. they are different icon. Is it the "merge" here indicates that it must be unique key only used for the Merge-Portal plugin? If so, is there anyway in the current implementation to detect/avoid this resource location clash?

Reply:
Nope.  If there's contention, the first WebContentService checked is the one that wins.  Detection wouldn't be terribly hard, but there would be a slight performance hit to keep checking after the first match.